arboretum/kubernetes/reports/compliance_oscal_observations.py

```diff
@@ -0,0 +1,227 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The compliance OSCAL observations report.
+
+A json report comprising NIST OSCAL Assessment Results Observations generated
```

Comment:
The use of "comprising" here is confusing to me.  Could this be rewritten a bit?

Reply:
Replaced "comprising" with "containing".